packages/cli/src/commands/dev.js

```diff
@@ -41,13 +45,14 @@ export const handler = async ({ app = ['db', 'api', 'web'] }) => {
         'web'
       )} && yarn webpack-dev-server --config ../node_modules/@redwoodjs/core/config/webpack.development.js`,
       prefixColor: 'blue',
+      runWhen: () => true,
     },
   }
 
   concurrently(
     Object.keys(jobs)
       .map((n) => app.includes(n) && jobs[n])
-      .filter(Boolean),
+      .filter((job) => job.runWhen()),
```

Comment:
We have the ability to only run the api or web sides using the following commands:
`yarn rw dev api`
`yarn rw dev web`

This seems to remove that ability, I think you could reintroduce it by adding the first boolean, and then filtering again by `runWhen`.

It would be great if the `web` side was also not run if it didn't exist.

Reply:
You're right @peterp ; just pushed an update for this.

I've tried to keep the code succinct, but let me know if you prefer having two calls to filter - `.filter(Boolean).filter(job => job.runWhen())` - for expressiveness

Also made it so the web side won't run unless the web folder exists